hermes_core/util/util.py

```diff
@@ -117,37 +122,65 @@ def parse_science_filename(filename):
         "descriptor": None,
     }
 
-    filename_components = filename.split("_")
+    file_name, file_ext = os.path.splitext(filename)
 
-    if (
-        filename_components[0] != hermes_core.MISSION_NAME
-        or filename_components[1] not in hermes_core.INST_SHORTNAMES
-    ):
-        raise ValueError("File {} not recognized.".format(filename))
+    filename_components = file_name.split("_")
 
-    #  reverse the dictionary to look up instrument name from the short name
-    from_shortname = {v: k for k, v in hermes_core.INST_TO_SHORTNAME.items()}
+    if filename_components[0] != hermes_core.MISSION_NAME:
+        raise ValueError(
+            "File {} not recognized. Not a valid mission name.".format(filename)
+        )
 
-    result["instrument"] = from_shortname[filename_components[1]]
-    result["version"] = filename_components[-1][1:]  # remove the v
-    result["time"] = Time.strptime(
-        filename_components[-3] + "_" + filename_components[-2], TIME_FORMAT
-    )
+    if file_ext == ".bin":
```

Comment:
So I've tested this with the different `l0` instrument files synced from the MOC, I did notice a file structured like: `hermes_CFDP_TEST_l0_2022260-030001_v01.bin` and wonder if we should add the ability to parse whether or not we parse this file as a test file unless the instrument files won't ever have that value `TEST`.

Reply:
Let's create an issue on this and save this for later. We need to understand how these files might be used. Maybe the SOC should ignore them.